### DISCUSSION TITLE
klMinecraft plugin

### DISCUSSION THREAD

**Discussion by BRlayer** [community/community#120910] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The plugin for Minecraft, Crates+,  don't work with users that they don't have OP, so.

How I can configure the crates for the plugin works for everyone?

**Comment by ebndev** [comment]
Hi @BRlayer,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Minecraft support. GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
